### PR TITLE
General Cleanup Part 1

### PR DESCRIPTION
**Pull request by Aathish04**
I started cleaning some stuff up as we previously planned in the Order of Business Document.

Nothing too drastic - this shouldn't impact moving from Heroku to Replit for testing at all.

Stuff that's been done, in decreasing order of importance, followed by the relevant commit:
1. Updated `package.json` to be more up to date. 1c64d1a0fa013837081bb9bfcf554f4f61002eb7
2. Got rid of `config.json` in favour of environment variables and `AppConfig.js`. a0cfd94fb8d5d986dfc46838601ccafc9060f024
3. Get rid of `notes.txt` - I moved the stuff mentioned there to the Order of Business Document. afd916942b5bf92a5d2e0fe3916285f2c0165aed
4. Got rid of the `cmds` folder that was never used except in a comment. ec1a36918b3af5924ef7927823daf682ac6a9dc4
5. Changed the default command prefix to `rub`. Yes it's a stupid genie joke but it works, no? c662ce918203d57980d8341a4b580147a9d61203

### CODE REVIEW

File: cmds/economy/balance.js [hunk 1]
```diff
@@ -1,45 +0,0 @@
...
-        const target = message.mentions.users.first() || message.author;
-        const targetId = target.id;
-        const name = target.username;
-        const result = await getUser(name, targetId);
-        let desc = `coins:\t **${result.coins}**`;
-        desc += `\nvault:\t **${result.vault_coins}/${result.vault_size}**`;
-        const embed = new Discord.MessageEmbed().setTitle(`${name}'s richness`).setDescription(desc);
-        //message.channel.send(`<@${targetId}> has **${coins}** coins`);
-        message.channel.send(embed);
-    }
```
Comment (vrishinvv) [inline cmds/economy/balance.js:44]: This is a newer, more supported way of creating commands, in some areas even better
so lets revert this deletion commit - this can help in the future, if we want to move to this patter of creating commands

File: index.js [hunk 2]
```diff
@@ -37,14 +37,6 @@ const client = new Commando.CommandoClient({
...
 
          await mongo();
-         /* client.registry
-            .registerGroups([
-                ['moderation', 'moderation commands'],
-                ['misc', 'misc commands'],
-                ['economy', 'money and economy system commands'],
-            ])
-            .registerDefaults()
-            .registerCommandsIn(path.join(__dirname, 'cmds')); */
```
Comment (vrishinvv) [inline index.js:47]: lets revert this change - same reason as above

**Review by vrishinvv — COMMENTED** [review]
Left comments - please check

**Comment by Aathish04** [comment]
> Left comments - please check

It seems we'll be rewriting most of the bot from scratch anyway, making this PR redundant. Self-closing now!
